src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -208,7 +209,16 @@ public void cancelLoading() {
     }
 
     /**
-     * The layout to display in the tab when it's loading
+     * The
+     * layout
+     * to
+     * display
+     * in
+     * the
+     * tab
+     * when
+     * it's
+     * loading
      */
```

Comment:
Are you sure, comments are supposed to look like this? IMHO, it is hard to read.

Reply:
Nope, thanks, that's the wrong formatting of intellilj